nevergrad/functions/functionlib.py

```diff
@@ -63,6 +63,96 @@ def _short_repr(self) -> str:
         return "Photonics"
 
 
+class PBT_no_overfitting(ExperimentFunction):
```

Comment:
Please rename this to the correct case. Probably `NoOverfittingPBT`
Also, I would move this to a different file, this is too different from other cases

Reply:
Done.